cmd/eksctl-anywhere/cmd/importimages.go

```diff
@@ -47,15 +51,29 @@ var importImagesCmd = &cobra.Command{
 	},
 }
 
-func importImages(context context.Context, spec string) error {
+func importImages(ctx context.Context, spec string) error {
+	registryUsername := os.Getenv("HELM_REGISTRY_USERNAME")
```

Comment:
Using helm in the name feels a bit wierd. The user is running a import images command, which we are also importing s helm chart, but they don't necessarily know that. We just happen to be using their registry for the helm chart too.

Reply:
yeah, changed this to REGISTRY_USERNAME/REGISTRY_PASSWORD